modules/contact-form/grunion-contact-form.php

```diff
@@ -2423,6 +2423,21 @@ function process_submission() {
 
 		update_post_meta( $post_id, '_feedback_extra_fields', $this->addslashes_deep( $extra_values ) );
 
+		$form_data = array();
+		foreach ( $this->fields as $key => $field ) {
+			$form_data[ $key ] = array(
+				'key'     => $key,
+				'value'   => $field->value,
+				'type'    => $field->attributes['type'],
+				'label'   => $field->attributes['label'],
+				'options' => isset( $field->attributes['options'] ) ? $field->attributes['options'] : '',
+				'post_id' => $post_id,
```

Comment:
Do we need to store the $post_id here?
I am not sure how that post ID is different from the feedback post Id that gets created?

Reply:
This post id is the same as the post id of the feedback post that gets created, and that's the point. The post_id in this array is how we can look up all of the feedback fields in ES associated with a particular post if we wanted to.

Reply:
We could just add that on the ES side if really need be, there is no need to send that info in the array since we already send the post_id when we add the post meta.